aten/src/ATen/native/cpu/BlasKernel.cpp

```diff
@@ -110,35 +108,6 @@ void gemm_transa_(
   }
 }
 
-template <typename scalar_t, typename opmath_t>
-void gemm_transb_(
```

Comment:
Why are you removing these kernels entirely, rather than specializing `gemm_core` for Half/BFloat16?

Reply:
I thought `transb_` and `notrans_` can be unified into one kernel so I removed it. But as you found before, I was read the wrong dimension for `transb_` case, so I added it back.
The `opmath` is introduced to solve `Half/BFloat16`, so I guess I do not need to specializing `gemm_core` for Half/BFloat16 then.